src/main/resources/db_scripts/postgres-rutherford-create-script.sql

```diff
@@ -176,7 +176,8 @@ CREATE TABLE event_bookings (
     user_id integer NOT NULL,
     status text DEFAULT 'CONFIRMED'::text NOT NULL,
     updated timestamp without time zone,
-    additional_booking_information jsonb
+    additional_booking_information jsonb,
+    attended boolean

```

Comment:
Do you think we need this or shall we just have another status enum entry that is attended instead of CONFIRMED?

Reply:
Ah yes, you're right. I had thought of doing that the other week when I first thought about adding this functionality, just didn't remember when it came to adding it yesterday.